### DISCUSSION TITLE
"Failed to Fetch" on Chrome Extension

### DISCUSSION THREAD

**Discussion by wjziv** [dani-garcia/vaultwarden#1686] [Q&A] [answered]
I'm having issues connecting to Vaultwarden through the Chrome extension on a particular machine. All others I've tested with work fine. Any browser, any extension, any OS, any machine has worked just fine.

On this one, single machine, I can log into the website and manipulate my account through the web app, but the Chrome extension flags a "Failed to Fetch" error and cannot log in.

I've tried the account I initially found hit the errors on other devices extensions/web app - No issues.
I've tried the device with other accounts on the web app - No issues.
I've tried the device with other accounts on the extension - Same issue.

So I've narrowed the problem down to extensions on a single machine, but for the life of me, I can't diagnose it.

Vaultwarden is running in a Docker container using Caddy and a self-signed certificate.

**Comment by wjziv** [comment, marked as answer]
Returning to report the issue: my host server URL was misspelled.

PEBCAK 100%. Hopefully no one else has this problem.

**Comment by imKeyurdesai** [comment]
Thanks for the solution! I literally had the exact same mistake lol
